editor/editor_properties.cpp

```diff
@@ -1267,16 +1267,20 @@ void EditorPropertyRect2::setup(double p_min, double p_max, double p_step, bool
 
 EditorPropertyRect2::EditorPropertyRect2(bool p_force_wide) {
 	bool horizontal = p_force_wide || bool(EDITOR_GET("interface/inspector/horizontal_vector_types_editing"));
-
+	bool grid = false;
```

Comment:
What is the point of `grid`? The only time you change it to true is when `horizontal == true`. Therefore `grid == horizontal` always. So why not just use `if (horizontal)` rather than `if (grid)` ?

Reply:
If both `p_force_wide` and `horizontal` are true, `grid` will be false. In that case the inspector would crash. Well I could change the condition to `if (horizontal && !p_force_wide)`, but it's safer this way. Basically my idea was that "if `grid` is true, `bc` has 2 sub-containers".

Reply:
Ok so `p_force_wide` makes it 1 row, 4 columns, `horizontal` makes it 2r 2c and neither makes it 4r 1c. Is that right?

Reply:
Yeah.

Reply:
Ok that makes sense.